### ISSUE TITLE
use of undeclared crate or module `jpeg_encoder`

### ISSUE THREAD

**Issue by twirrim** [jay3332/ril#15]
I've been using ril to generate some PNGs, thought I'd see what'd happen if I switched to GIFs, so I added the "gif" feature alongside the "png" one already enabled.  The result is that ril won't compile due to a missing dependency.   Adding the "jpeg" features fixes it as that then pulls in the needed crates.  I assume you've got an unintended side effect somewhere breaking some of that isolation.

```
   Compiling ril v0.9.0
error[E0433]: failed to resolve: use of undeclared crate or module `jpeg_encoder`
   --> /home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/ril-0.9.0/src/error.rs:161:11
    |
161 | impl From<jpeg_encoder::EncodingError> for Error {
    |           ^^^^^^^^^^^^ use of undeclared crate or module `jpeg_encoder`

error[E0433]: failed to resolve: use of undeclared crate or module `jpeg_encoder`
   --> /home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/ril-0.9.0/src/error.rs:162:18
    |
162 |     fn from(err: jpeg_encoder::EncodingError) -> Self {
    |                  ^^^^^^^^^^^^ use of undeclared crate or module `jpeg_encoder`

error[E0433]: failed to resolve: use of undeclared crate or module `jpeg_encoder`
   --> /home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/ril-0.9.0/src/error.rs:164:13
    |
164 |             jpeg_encoder::EncodingError::IoError(err) => Self::IOError(err),
    |             ^^^^^^^^^^^^ use of undeclared crate or module `jpeg_encoder`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `ril` due to 3 previous errors
```

**Comment by jay3332** [comment]
I believe this was fixed in #12, but the change has not yet been uploaded to crates.io

**Comment by jay3332** [comment]
uploaded!
